Reorganize the XO's office, give heads tablets

babydoll opened the pull request:
This reorganizes the XO's office. Please see attached pictures.

This also adds modular laptops to every head of staff's office, replacing the full-size consoles which were added in #18451. The CO, XO and RD get the 'advanced' variant and every other head of staff gets the standard variant.

Additionally adds special pens for the CO and XO, icon states for many research department areas, and a new carpet, used on the bridge.

### XO's office (map editor view)
(new pen visible next to the console; captain's is gold. tablet not added in this screenshot, but it's on the same table as the new pen)
![](https://i.imgur.com/LGNEh6m.png)
### New carpet in-game 
(brown chairs have since been replaced with black ones to fit better)
![](https://i.imgur.com/0Cg3mog.png)

babydoll committed:
Reorganize the XO's office, give heads laptops

This reorganizes the XO's office. Please see attached pictures.

This also adds modular laptops to every head of staff's office,
replacing the full-size consoles which were added in #18451.
The CO, XO and RD get the 'advanced' variant and every other
head of staff gets the standard variant.

Additionally adds special pens for the CO and XO, icon states for many
research department areas, and a new carpet, used on the bridge.

babydoll commented:
Considering replacing the laptops with tablets. Thoughts?

comma commented:
I think laptops have more stuff and longer laasting battery. They can buy tablets if they want one.

HeyBanditoz commented:
Doesn't the XO's locker contain spare IDs and PDAs? Why is it in their quarters?

babydoll commented:
So they can change clothes in private, and to make room in their main office.

atlantiscze commented on maps/torch/torch-5.dmm:
```diff
@@ -51,25 +51,21 @@
+"bc" = (/obj/structure/closet/secure_closet/XO,/obj/machinery/atmospherics/unary/vent_scrubber/on{dir = 8},/obj/machinery/light{icon_state = "tube1"; dir = 4},/turf/simulated/floor/carpet/blue2,/area/crew_quarters/heads/office/xo)
+"bd" = (/obj/item/device/radio/intercom{pixel_y = 22},/obj/machinery/camera/network/command{c_tag = "CO - Office"},/turf/simulated/floor/carpet/blue2,/area/crew_quarters/heads/office/co)
+"be" = (/obj/machinery/light{icon_state = "tube1"; dir = 4},/obj/structure/bed/chair/comfy/black{tag = "icon-comfychair_preview (NORTH)"; icon_state = "comfychair_preview"; dir = 1},/turf/simulated/floor/carpet/blue2,/area/crew_quarters/heads/office/co)
+"bf" = (/obj/structure/table/steel,/obj/item/modular_computer/laptop/preset/custom_loadout/standard,/turf/simulated/floor/tiled,/area/crew_quarters/heads/office/ce)
```
Rather than custom_loadout/standard i'd recommend either using the existing /preset/records

Alternatively you can create a new one, /preset/heads or similar, with pre-installed programs. custom_loadout/standard doesn't have that.

(just as a note: this is not a change request, just a recommendation)

babydoll replied:
This is the one that spawns on the bridge (and is in the loadout) but I can see about making the new type

Hubblenaut commented:
Regarding laptops and tablets, they are available on loadout if the player wants to invest the points in them to have it as a Head of Staff. I don't think there is a need to map them in.

Remap looks good, except for that EFTPOS device spawning on the fax machine. I don't think an EFTPOS device even finds usage right now for anyone but the Research Director. Even if some few players can think of uses for it, it doesn't warrant blocking the fax machine with it IMO.

Also that cup is going to spawn offsite the table surface half the time if not given pixel offsets. Most other mapped cups have working offsets that are the same that can be achieved by the table item grid, might want to look at them.

babydoll commented:
I'm going to replace the laptops with full size consoles, and give tablets instead.

I've removed the EFTPOS (it said HoP anyway and I've literally never seen it used) and made sure the cup sits on the table correctly.

babydoll committed:
Reorganize the XO's office, command tablets

This reorganizes the XO's office. Please see attached pictures.

This also adds modular laptops to every head of staff's office,
replacing the full-size consoles which were added in #18451.
The CO, XO and RD get the 'advanced' variant and every other
head of staff gets the standard variant.

Additionally adds special pens for the CO and XO, icon states for many
research department areas, and a new carpet, used on the bridge.

babydoll commented:
Updates in most recent commit:

- Added modular consoles (command preset) to head offices
- Added modular tablets with new command preset and unique icon (no stealing!) to offices (BO's are in their lockers)
- The laptop that was added so you could use a computer while sitting at the 'records station' has been removed
- Moved some table clutter around in the offices to fit the consoles better
- Condensed the two ID boxes in the XO's locker to one box/large
- Updated changelog

babydoll commented:
Travis realized the error of his ways. Final revision for this PR, at least, assuming reviewer agrees.

atlantiscze commented:
@sabiram Conflicts.

babydoll committed:
Merge branch 'dev' of https://github.com/Baystation12/Baystation12 into xoxoxo

# Conflicts:
#	code/modules/modular_computers/computers/subtypes/preset_laptop.dm
#	code/modules/modular_computers/computers/subtypes/preset_tablet.dm
#	icons/obj/modular_tablet.dmi

babydoll commented:
Conflicts resolved

babydoll commented:
Someone asked so here's an image of the captain's pen and the tablet.
![](https://i.imgur.com/7Ly4dAm.png)
